contest-service/service.bal

```diff
@@ -442,6 +484,50 @@ service /contestService on new http:Listener(9098) {
         }
     }
 
+    resource function post contests/[string contestId]/register/[string userId]() returns string|http:InternalServerError|http:BadRequest {
```

Comment:
I would use `registrants` as the resource name here. It matches with the get resource name.